.github/workflows/track-acvm.yml

```
@@ -0,0 +1,28 @@
+name: Track ACVM for Integration Tests
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  check_matching_version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Check acvm version change
+        run: |
+          # Extract acvm version from the current Cargo.lock
+          CURRENT_VERSION=$(awk '/name = "acvm"/ {getline; print $3}' Cargo.lock | tr -d '"')
+          INTEGRATION_TEST_VERSION=$(jq '.dependencies["@noir-lang/acvm_js"]' compiler/integration-tests/package.json | tr -d '"' | sed 's/^[^0-9]*//')
```

```suggestion
          INTEGRATION_TEST_VERSION=$(yarn workspace integration-tests info @noir-lang/acvm_js --json | jq .children.Version | tr -d '"')
```

This should be equivalent but doesn't require me to think too hard about regexes.